source/server/infrastructure/commit_publisher.coffee

```diff
@@ -29,37 +28,65 @@ class Space.eventSourcing.CommitPublisher extends Space.Object
           update: $push: { receivers: { appId: appId, receivedAt: new Date() } }
         })
         # Only publish the event if this process was the one that locked it
-        @publishCommit(@_parseCommit(lockedCommit)) if lockedCommit?
+        @publishChanges(@parseChanges(lockedCommit), commit._id) if lockedCommit?
     }
+    @log.info @_logMsg("observing for commits via the collection")
 
   stopPublishing: ->
-    @log.info @_logMsg("stopped")
     @_publishHandle?.stop()
+    @log.info @_logMsg("collection observer stopped")
```

Comment:
Wanted to make this more explicit, as stopping the publisher only stops the observer and not any publishing of commits generated in the same process! We could always use the state to just ignore the call to publish the commit when called by the `commitStore` too though. Just depends on if it makes sense to have control over publishing in general, or if it's only relevant to externally generated commits.

cc @meteor-space/core